linux/src/reals_linux/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(azure_c_util_reals_c_files
```

Comment:
>azure_c_util_reals_c_files [](start = 4, length = 26)

This folder (reals_linux) belongs in linux/tests/reals_linux, just like we did for win32. Should also have an entry in the CMakeLists.txt under linux/tests that references it.